### PR TITLE
Fix python2 problems

### PR DESCRIPTION
**Pull request by lesteve**
Took the Python 2-related part of #29 with some minor tweaks.

@guillaumeeb let me know if I forgot anything!

### CODE REVIEW

**Comment by lesteve** [comment]
So the CI is not going to tell us anything about the Python 2 fixes but I feel this is better to fix our Python 2 testing in a separate PR.

File: dask_jobqueue/pbs.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from __future__ import division
+
```
Comment (guillaumeeb) [inline dask_jobqueue/pbs.py]: I use some division in slurm.py too. You took it in pbs.py, but not in slurm.py, is this intended?
I can see you've imported that also in the __init__.py file, I'm not sure what it does, and if it is enough for both pbs and slurm then (not coming from pythonic world).

**Comment by lesteve** [comment]
So annoyingly test_sge.test_basic seems to fail quite consistently ... @guillaumeeb in your experience did you find that increasing the timeout helps?

**Comment by guillaumeeb** [comment]
I would not talk about experience, but in my unique try with travis and a timeout of 20 seconds, tests succeeded...
I know that is not particularly helpful.

**Comment by lesteve** [comment]
OK that is the kind of thing I wanted to hear about, so that is still helpful thanks!

**Comment by lesteve** [comment]
An alternative is to wait for https://github.com/Chilipp/docrep/pull/9 to be merged.

I don't feel strongly about how much we care about class docstrings correctness for Python 2.

**Comment by guillaumeeb** [comment]
You're a lot more experienced that I am in this area. I don't know what might be the consequences of non standard docstring use like in pbs.py...

**Comment by mrocklin** [comment]
I ran into this in practice with someone who was using Python 2.  I noticed that they were using some of the parameters to the PBSCluster but not others (they were missing threads, processes, ...)  We pulled up the docstring and I noticed that they weren't included.

A low-tech but maybe sufficient solution to this is just to drop docrep and copy-paste everywhere.  This might not be terrible.

**Comment by lesteve** [comment]
I have to say, I am a bit undecided about `docrep` ... but let's keep this PR about Python 2. I am going to merge this one.

**Pull request merged**